javascripts/verify-member.js

```diff
@@ -0,0 +1,34 @@
+var verifyMemberApp = angular.module('member-verify', []);
+
+verifyMemberApp.controller('VerifyCtrl', ['$scope', 'validateMember', function VerifyCtrl($scope, validateMember) {
+    var eChk = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Eewww... could you break this down in comments, please? I don't think anyone, including future us, wants to try and figure out what this is doing. It almost looks like this is an email checking, but seems like there's too many '@'.